stories/toast/toast.stories.mdx

```diff
@@ -0,0 +1,151 @@
+import { Canvas, Meta, Preview, Story } from "@storybook/addon-docs";
+import { Notification } from "src/toast";
+import { Text } from "src/text";
+import { Secondary, Heading3, Title } from "../storybook-common";
+import { PropsTable } from "./props-table";
+
+<Meta title="Modules/Toast" component={Notification} />
+
+<Title>Notification Toast</Title>
+
+<Secondary>Overview</Secondary>
+
+<Text.BodySmall>
+    <code>Notification toast</code> are used to inform users of updates or
+    changes to system status. Communicating with users and providing immediate
+    feedback are important for building trust. While notifications are an
+    effective method of communicating with users, they are disruptive and should
+    be used sparingly.
+</Text.BodySmall>
+
+```tsx
+import { Toast } from "@lifesg/react-design-system/toast";
+```
+
+<Heading3>Light theme notification toast</Heading3>
+
+<Text.BodySmall>
+    Specifying the <code>light.theme</code> interfaces will have been optimized
+    for the standard light mode.
+</Text.BodySmall>
+
+<Preview>
+    <Story name="Notification toast">
+        <Text.H4>Success notification toast in context</Text.H4>
```

Comment:
our practice is for the descriptions to sit outside of the story to make the story focused. Refer to how the other stories are being done.

Reply:
And what does it mean by "in context"?